src/aui/tabart.cpp

```diff
@@ -1192,11 +1204,12 @@ wxSize wxAuiSimpleTabArt::GetTabSize(wxDC& dc,
     dc.SetFont(m_measuringFont);
     dc.GetTextExtent(caption, &measured_textx, &measured_texty);
 
-    wxCoord tab_height = measured_texty + 4;
-    wxCoord tab_width = measured_textx + tab_height + 5;
+    wxCoord tab_height = measured_texty + wnd->FromDIP(4);
+    wxCoord tab_width = measured_textx + tab_height + wnd->FromDIP(5);
 
     if (close_button_state != wxAUI_BUTTON_STATE_HIDDEN)
-        tab_width += m_activeCloseBmp.GetScaledWidth();
+        // increase by button size plus the padding
```

Comment:
Trivial, but could you please add `{` and `}` around the (now) multiline `if` statement body?

Reply:
Yes, I added that.

Reply:
This doesn't seem to have been done?